test/faker-lib.js

```diff
@@ -0,0 +1,12 @@
+const faker = require('faker');
+module.exports = { 
+    quiz_name : faker.name.firstName(),
+    description : faker.commerce.productDescription(),
+    prizeMoney : parseInt(faker.commerce.price()),
+    starttime : parseInt(faker.date.future().getTime()/1000),
+    time: parseInt(faker.date.past().getTime()/1000),
+    duration : 100,
+    questionHash : faker.git.commitSha(),
+    answerHash : faker.git.commitSha(),
+    currentTime : Date.now()
+};
```

Comment:
Change the file name or create a function called `create fake quiz` or something like that, so that we can know what this function is doing

Reply:
Add  a newline at end of file